### PR TITLE
Support for mkdocs material

### PR DESCRIPTION
**Pull request by avandierast**
Here is the implementation to support the pymdown superfences brace format.

**Issue:** https://github.com/modal-labs/pytest-markdown-docs/issues/23

Any comments are welcome :)

### CODE REVIEW

File: src/pytest_markdown_docs/plugin.py [hunk 1]
```diff
@@ -182,6 +182,26 @@ def extract_code_blocks(
             prev = code_block
 
 
+def parse_block_info(block_info: str) -> typing.List[str]:
+    block_info = block_info.strip()
+    # The default `python continuation` format is not compatible with Material for Mkdocs.
+    # But, PyMdown Superfences has a special brace format to add options to code fence blocks: `{.<lang> <option1> <option2>}`.
+    if block_info.startswith("{"):
+        block_info = block_info.strip("{").strip("}")
```
Comment (RafaelWO) [inline src/pytest_markdown_docs/plugin.py:190]: You can specify multiple chars within `str.split()` and thus only call it once 😉 
```suggestion
        block_info = block_info.strip("{}")
```

**Review by RafaelWO — COMMENTED** [review]
Thanks for your work! 😃 As I am interested in this change I am curious whether you can confirm that this works with mkdocs and the additional attributes don't mess up the classes added to the resulting HTML code?

**Comment by avandierast** [comment]
> As I am interested in this change I am curious whether you can confirm that this works with mkdocs and the additional attributes don't mess up the classes added to the resulting HTML code?

I'll check that :)

**Comment by avandierast** [comment]
It seems to work pretty well :)

With the following fence config: `{.python continuation .extra-class #id hl_lines="1 3"}`.
I get this in the HTML code: `<div id="id" class="language-python extra-class highlight">`.

**Comment by avandierast** [comment]
@freider I've added the "--markdown-docs-syntax=superfences" as requested :)

**Review by freider — APPROVED** [review]
Thanks for the contribution! Lgtm

**Pull request merged**